src/core/settings/qgssettingsregistrycore.cpp

```diff
@@ -150,10 +150,37 @@ void QgsSettingsRegistryCore::migrateOldSettings()
   pal::Pal::settingsRenderingLabelCandidatesLimitLines->copyValueFromKey( QStringLiteral( "core/rendering/label_candidates_limit_lines" ), true );
   pal::Pal::settingsRenderingLabelCandidatesLimitPolygons->copyValueFromKey( QStringLiteral( "core/rendering/label_candidates_limit_polygons" ), true );
 
+  // handle bad migration - Fix profiles for old QGIS versions (restore the Map/scales key on windows)
```

Comment:
can we maybe add a note to drop this when 3.34 is released (which becomes next LTR)?

Reply:
Is it safe to assume no one will upgrade from 3.30.0|1 to 3.34.X though?

Reply:
you're right, bump it to 3.36, so 3.34 is LTR and 3.28 is not supported anymore.
and....if the user could live for 1 year with the bug, I'd say it's ok to drop backward compatibility of an unsupported version. It's not a big deal, but that code is not really explicit, so I'd rather avoid keeping it. Alaso a link to this PR might help too.